ClientLib/src/main/java/com/o3dr/services/android/lib/drone/action/GuidedActions.kt

```diff
@@ -0,0 +1,7 @@
+package com.o3dr.services.android.lib.drone.action
+
+/**
+ * Created by Fredia Huya-Kouadio on 1/19/15.
+ */
+class GuidedActions  //Private to prevent instantiation
```

Comment:
Not sure what the purpose this class serves. It's not referenced anywhere else in the project currently

Presumably the intention was to house constant Strings like this other Action classes, in which case we should make it an `object GuidedActions  {}` instead of a class

Reply:
Yeah, I'm in the mode of "refactoring tons of stuff to Kotlin" and making sure everything continues to work. At some point I'll circle back and address more trivial stuff like this. My guess is that `GuidedActions` was probably a placeholder put in place by the original author. I'm not sure what it does, and will investigate at some point. If there's no need for it, I'll take the appropriate action then.